source/material_model/interface.cc

```diff
@@ -747,6 +747,46 @@ namespace aspect
       return vs;
     }
 
+    namespace
+    {
+      std::vector<std::string> make_plastic_additional_outputs_names()
+      {
+        std::vector<std::string> names;
+        names.push_back("weakened_cohesions");
+        names.push_back("weakened_friction_angles");
+        return names;
+      }
+    }
+
+    template<int dim>
+    PlasticAdditionalOutputs<dim>::PlasticAdditionalOutputs (const unsigned int n_points)
+      :
+      NamedAdditionalMaterialOutputs<dim>(make_plastic_additional_outputs_names()),
+      cohesions(n_points, -1.0),
+      friction_angles(n_points, -1.0)
```

Comment:
Please replace the -1 by `numbers::signaling_nan<double>()` in both lines. -1 was a leftover from early days in the seismic velocities, and we would like to get rid of it.